staging/src/k8s.io/apiserver/pkg/authorization/rule/rule.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rule
+
+// DefaultResourceInfo holds information that describes a rule for the resource
+type DefaultResourceInfo struct {
```

Comment:
Any objections with just putting this into the `authorization` package? Same as `Attribute` is referenced by `Authorizer` and in the same package.